AmericanMedicalAssociation/ama-style-guide-2
# EWL-8904: Styling and styleguide page for content type list view.

## Ticket(s)
**Do not** submit a Pull Request without a relevant JIRA ticket or Github issue! If you are creating a new pattern or feature, create an issue describing the need for this feature in Github **first** and then link to it below.

**Github Issue**
- N/A

**Jira Ticket**
- [EWL-8904: Video Content Type index Page - Styleguide](https://issues.ama-assn.org/browse/EWL-8904)

## Description
Created styleguide page and styling for the new content type listing view. PR for main ticket [(EWL-8889)](https://issues.ama-assn.org/browse/EWL-8899) is here: [https://github.com/AmericanMedicalAssociation/ama-d8/pull/2802](https://github.com/AmericanMedicalAssociation/ama-d8/pull/2802)

## To Test
- (in appropriate styleguide folder) Run 'gulp serve' to open styleguide
- Under Pages select Content Type Listing
- Confirm styling matches the following zeplins:
Zeplins: 
[Desktop](https://zpl.io/V0roklx)

(Note that no designs for tablet or mobile were provided, use the series index page designs for reference)
[Tablet](https://zpl.io/bW8XJej)
[Mobile](https://zpl.io/2p3Pe5y)

- Follow the instructions in the following PR to get a local instance of the Content Type view:  [https://github.com/AmericanMedicalAssociation/ama-d8/pull/2802](https://github.com/AmericanMedicalAssociation/ama-d8/pull/2802)
- In local, run lando link-styleguides to get latest styling
- Confirm the styling of the page matches the above zeplins

## Visual Regressions
- N/A

## Relevant Screenshots/GIFs
![Screen Shot 2021-07-07 at 4 38 15 PM](https://user-images.githubusercontent.com/67962801/124832632-8a88f880-df42-11eb-914c-ae92d201c563.png)
![Screen Shot 2021-07-07 at 4 38 34 PM](https://user-images.githubusercontent.com/67962801/124832634-8b218f00-df42-11eb-879f-e1891df85afd.png)

## Remaining Tasks
- N/A

## Additional Notes
- N/A

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)
